flag_bool_with_inverse_test.go

```diff
@@ -334,10 +334,87 @@ func TestBoolWithInverseNames(t *testing.T) {
 	require.Len(t, names, 2)
 	require.Equal(t, "env", names[0], "expected first name to be `env`")
 	require.Equal(t, "no-env", names[1], "expected first name to be `no-env`")
+}
+
+func TestBoolWithInverseString(t *testing.T) {
+	tcs := []struct {
+		testName      string
+		flagName      string
+		required      bool
+		usage         string
+		inversePrefix string
+		expected      string
+	}{
+		{
+			testName: "empty inverse prefix",
+			flagName: "",
+			required: true,
+			expected: "--[no-]\t",
+		},
+		{
+			testName: "single-char flag name",
+			flagName: "e",
+			required: true,
+			expected: "-[no-]e\t",
+		},
+		{
+			testName: "multi-char flag name",
+			flagName: "env",
+			required: true,
+			expected: "--[no-]env\t",
+		},
+		{
+			testName: "required with usage",
+			flagName: "env",
+			required: true,
+			usage:    "env usage",
+			expected: "--[no-]env\tenv usage",
+		},
+		{
+			testName: "required without usage",
+			flagName: "env",
+			required: true,
+			expected: "--[no-]env\t",
+		},
+		{
+			testName: "not required with default usage",
+			flagName: "env",
+			required: false,
+			expected: "--[no-]env\t(default: false)",
+		},
+		{
+			testName:      "custom inverse prefix",
+			flagName:      "env",
+			required:      true,
+			inversePrefix: "nope-",
+			expected:      "--[nope-]env\t",
+		},
+		{
+			testName:      "empty inverse prefix",
+			flagName:      "env",
+			required:      true,
+			inversePrefix: "",
+			expected:      "--[no-]env\t",
+		},
+	}
 
-	flagString := flag.String()
-	require.Contains(t, flagString, "--env")
-	require.Contains(t, flagString, "--no-env")
+	for _, tc := range tcs {
+		t.Run(tc.testName, func(t *testing.T) {
+			flag := &BoolWithInverseFlag{
+				BoolFlag: &BoolFlag{
+					Name:     tc.flagName,
+					Usage:    tc.usage,
+					Required: tc.required,
+				},
+			}
```

Comment:
```suggestion
      InversePrefix: tc.InversePrefix,
	}
```
This will remove the test coverage warning above !!!!